src/userInterface/util/enums/MergeModes.ts

```diff
@@ -0,0 +1,33 @@
+export enum MergeModes {
+    diff = 'diff',
+    reload = 'reload',
+    save = 'save'
+}
+
+export interface MergeText {
```

Comment:
keep this in the userInterFace/util/interfaces or types

Reply:
Done